app/src/main/java/com/x8bit/bitwarden/ui/platform/components/scaffold/BitwardenScaffold.kt

```diff
@@ -33,6 +36,8 @@ import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
 fun BitwardenScaffold(
     modifier: Modifier = Modifier,
     topBar: @Composable () -> Unit = { },
+    utilityBar: @Composable () -> Unit = { },
```

Comment:
could we add KDoc for these two since they are not just 1:1 with existing material pieces? I do really like the inclusion and how we are using them!